src/components/MessageInput/hooks/useCooldownTimer.tsx

```diff
@@ -53,29 +41,32 @@ export type CooldownTimerState = {
 export const useCooldownTimer = <
   StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
 >(): CooldownTimerState => {
-  const { client } = useChatContext<StreamChatGenerics>('useCooldownTimer');
+  const { latestMessageDatesByChannels } = useChatContext<StreamChatGenerics>('useCooldownTimer');
   const { channel } = useChannelStateContext<StreamChatGenerics>('useCooldownTimer');
 
-  const { cooldown: cooldownInterval } = (channel.data ||
+  const { cooldown: cooldownInterval, own_capabilities } = (channel.data ||
     {}) as ChannelResponse<StreamChatGenerics>;
 
   const [cooldownRemaining, setCooldownRemaining] = useState<number>();
 
-  const globalRole = client.user?.role || '';
```

Comment:
I just checked documentation for own_capabilities and looks like these are at the channel level. Do we still need to account for global roles like moderator and admin?

Reply:
I think we don't because own_capabilities also takes [permissions](https://getstream.io/chat/docs/javascript/chat_permission_policies/?language=javascript) into account, which in my understanding contain global role

Reply:
ok! I will approve.